Description: python-imaging provides a Pillow egg
 On older Ubuntu series, the egg is named PIL.
Forwarded: not-needed
Index: schooltool/setup.py
===================================================================
--- schooltool.orig/setup.py
+++ schooltool/setup.py
@@ -66,3 +66,2 @@ Javascript will be usable, although perh
                       'PasteScript',
-                      'Pillow',
                       'pytz',
